.bundle/config

```diff
@@ -1,4 +1,3 @@
 ---
 BUNDLE_WITHOUT: omnibus_package
-BUNDLE_DISABLE_SHARED_GEMS: '1'
```

Comment:
Oh, I like this change - so it'll use the built-in `json` gem if it satisfies the requirements, right?